src/Model/ModelManager.php

```diff
@@ -339,6 +339,12 @@ public function getIdentifierValues($entity)
                 continue;
             }
 
+            if (method_exists($value, 'getId')) {
```

Comment:
There is no guarantee, that this method will return the primary key of your entity.

Reply:
You're right. I reviewed the function again and saw that what was introduced in December 2018 is incorrectly introduced. I will now perform the new PR.